101-bundle-json.md

```diff
@@ -441,8 +442,8 @@ The structure of a parameters section looks like the section below.
       "then": <json-schema>,
       "title": <string>,
       "type": <string> | [ <string> ],
-      "uniqueItems": boolean
-
+      "uniqueItems": boolean,
```

Comment:
should this be `"uniqueItems": <boolean>`

Reply:
Good catch! I've fixed that now.

Reply:
awesome thanks!